new/LIT7104Yegbaru.xml

```diff
@@ -0,0 +1,93 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7104Yegbaru" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይግበሩ፡ በኁባሬ፡ ተዝካረ፡ ሰንብት፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Yǝgbaru ba-ḫubare tazkāra sanbat (Qǝne of śǝllāse-type)</title>
```

Comment:
ḫubāre

Reply:
śǝllāse type (without hyphen as above)

Reply:
No. The proper orthography is with hyphen.